Jenkinsfile

```
@@ -1,6 +1,6 @@
 #!/usr/bin/env groovy
 
-@Library('caf-continuous-integration@topic/neverlord/extra-scripts') _
+@Library('caf-continuous-integration@topic/riemass/docker-workdir') _
```

Upstream PR is merged:

```suggestion
@Library('caf-continuous-integration') _
```